sources/editor/Stride.Assets.Presentation/AssetEditors/ProjectWatcher.cs

```diff
@@ -101,8 +104,31 @@ private async Task BatchChanges()
                     do
                     {
                         var assemblyChange = await buffer.ReceiveAsync(batchChangesCancellationTokenSource.Token);
+                        
+
                         if (assemblyChange != null)
+                        {
                             assemblyChanges.Add(assemblyChange);
+                            var project = assemblyChange.Project;
+                            var projects = msbuildWorkspace.CurrentSolution.GetProjectDependencyGraph().GetProjectsThatTransitivelyDependOnThisProject(project.Id);
+                            foreach(var p in projects)
+                            {
+                                var assemblyName = msbuildWorkspace.CurrentSolution.GetProject(p).AssemblyName;
+                                var target = trackedAssemblies.FirstOrDefault(x => x.Project.AssemblyName == assemblyName );
+                                if(target != null)
+                                {
+                                    string file = assemblyChange.ChangedFile;
+                                    if(assemblyChange.ChangeType == AssemblyChangeType.Binary)
+                                    {
+                                        file = target.LoadedAssembly.Path;
+                                    }else if(assemblyChange.ChangeType == AssemblyChangeType.Project)
```

Comment:
Formatting

Reply:
https://github.com/stride3d/stride/pull/2106/commits/46882e4cadb54636e246bd03cb94a7770e20a317
is it better with that one? or do you want somewhere else another formatting too?

Reply:
It is better now.